ramls/end-patron-action-session-requests.json

```diff
@@ -0,0 +1,23 @@
+{
```

Comment:
This JSON schema appears to be new, yet there appears to be no new reference. Were is it referenced?

Reply:
fixed